app/web/src/features/auth/signup/AccountForm.tsx

```diff
@@ -318,8 +318,9 @@ export default function AccountForm() {
           )}
         />
         <Typography variant="body1">
-          {SIGN_UP_TOS_TEXT}
-          <TOSLink />.
+          <Trans i18nKey="auth:account_form.tos_prompt">
+            To continue, please read and accept the <TOSLink />.
```

Comment:
Is the test here just ignored?

Reply:
It shouldn't be ignored as this is a template string for i18next to swap to the right component due to how weird this `Trans` component works, so not sure what you mean by this being ignored?

Reply:
Because there's English text here, but also under the auth:account_form.tos_prompt, so I thought maybe the text here doesn't get used, just the `<TosLink/>`. Of course I wouldn't change it - just wondering.

Reply:
Yeah you're right, the English text doesn't get displayed directly here, so it's not used in that respect, but it is used for determining the swap position of `<TOSLink />` in the JSON (or any React element if there are more) as the "template string". In this case, it'll read `<1></1>` in the JSON since it is the second children nested under `<Trans>` in this "template". 1 because it counts from 0.

Took me a few reading of this to understand how it works - https://react.i18next.com/latest/trans-component#how-to-get-the-correct-translation-string 😅